### PR TITLE
Subdomain

### PR DESCRIPTION
**Pull request by Marie59**
I thought a small news on the brand new earth-system subdomain could be nice :)

### CODE REVIEW

File: content/news/2023-10-17-earth-system/index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: "A brand new subdomain of Galaxy Europe : earth-system.usegalaxy.eu "
+date: "2023-10-17"
+authors: Marie Jossé
+tease: "A subdomain to process, analyse and access Earth System and Environmental data"
+hide_tease: true
+subsites: [all-eu,global]
```
Comment (bgruening) [inline content/news/2023-10-17-earth-system/index.md:7]:
```suggestion
tags: [esg-wp5, esg]
subsites: [all, all-eu, esg]
```

File: content/news/2023-10-17-earth-system/index.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+</div>
+
+Earth system models (ESM) seek to simulate all relevant aspects of the Earth system.
+ESMs are composed by a set of equations describing atmospheric and oceanic circulation and thermodynamics, but also the biological and chemical processes that feedback on to the physics of climate, all solved for a number of locations in space that form a three-dimensional grid over the surface of the Earth and underneath the surface of the oceans.
+
+
+
+### Fair-Ease : an EOSC project
+
+An approach to observation and modelling of the Earth System, Environment and Biodiversity.
```
Comment (bgruening) [inline content/news/2023-10-17-earth-system/index.md:23]: A link to the FAIR-Ease homepage?

File: content/news/2023-10-17-earth-system/index.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+- Coastal Water Dynamics : focuses on the coastal marine environment near river estuaries, where important processes take place.
+- Earth Critical Zone : monitors land and soil degradation.
+- Volcano Space Observatory : monitors global volcanic activity, allowing the focus on any major volcanic eruption worldwide.
+- Ocean Bio-Geochemical Observations: addresses questions regarding the health of marine ecosystems (e.g. ocean acidification, ...) and needs for ocean resource management.
+- Marine Omics : analyses of spatial- and time-comparable marine microbial metagenomics data sets for the exploration of biodiversity and its correlations with environmental quality.
+
+
+### Tools, tools and more tools
+
+Thanks to the help and partnership of Galaxy Europe and EuroScienceGateway project we were able to implement 6 interactive tools on usegalaxy.eu. There are now also available on [earth-system.usegalaxy.eu](https://earth-system.usegalaxy.eu/). Go running on the subdomain page to try out these tools !
```
Comment (bgruening) [inline content/news/2023-10-17-earth-system/index.md:40]: link to eurosciencegateway.eu?

Comment (bgruening) [reply content/news/2023-10-17-earth-system/index.md:40]:
```suggestion
Thanks to the help and partnership of Galaxy Europe and [EuroScienceGateway](https://eurosciencegateway.eu) project we were able to implement 6 interactive tools on usegalaxy.eu. They are now also available on [earth-system.usegalaxy.eu](https://earth-system.usegalaxy.eu/)! Go run them on [earth-system.usegalaxy.eu](https://earth-system.usegalaxy.eu/) and try these tools!
```

Comment (bgruening) [reply content/news/2023-10-17-earth-system/index.md:40]: Maybe a list of those tools? With links?

File: content/news/2023-10-17-earth-system/index.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+Don't hesitate to give use some feedbacks and help if you think they should be improve. As said before they are quite new and need to be tested and approved.
```
Comment (bgruening) [inline content/news/2023-10-17-earth-system/index.md:41]:
```suggestion
Don't hesitate to give us some feedback and help if you think they should be improved. As said before they are quite new and need to be tested and vetted by the community.
```

File: content/news/2023-10-17-earth-system/index.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+The final goal will be to have at least one workflow of tools to access, process and visualise data for each domain cited above. Hopefully even more than that !
```
Comment (bgruening) [inline content/news/2023-10-17-earth-system/index.md:43]:
```suggestion
One goal of FAIR-Ease is to have at least one workflow of tools to access, process, and visualise data for each domain cited above. Hopefully even more than that! 
```

File: content/news/2023-10-17-earth-system/index.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+- Marine Omics : analyses of spatial- and time-comparable marine microbial metagenomics data sets for the exploration of biodiversity and its correlations with environmental quality.
+
+
+### Tools, tools and more tools
+
+Thanks to the help and partnership of Galaxy Europe and EuroScienceGateway project we were able to implement 6 interactive tools on usegalaxy.eu. There are now also available on [earth-system.usegalaxy.eu](https://earth-system.usegalaxy.eu/). Go running on the subdomain page to try out these tools !
+Don't hesitate to give use some feedbacks and help if you think they should be improve. As said before they are quite new and need to be tested and approved.
+
+The final goal will be to have at least one workflow of tools to access, process and visualise data for each domain cited above. Hopefully even more than that ! 
+If you are interrested by earth system and environmental sciences and have tools that you think could fit right into earth-system.usegalaxy.eu, please contact us so we can try to work together on them ! The more the merrier !
```
Comment (bgruening) [inline content/news/2023-10-17-earth-system/index.md:44]:
```suggestion
If you are interested in earth-system and environmental sciences and have tools that you think could fit right into [earth-system.usegalaxy.eu](https://earth-system.usegalaxy.eu/), please contact us so we can try to work together! The more the merrier! 
```

File: content/news/2023-10-17-earth-system/index.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+### Tools, tools and more tools
+
+Thanks to the help and partnership of Galaxy Europe and EuroScienceGateway project we were able to implement 6 interactive tools on usegalaxy.eu. There are now also available on [earth-system.usegalaxy.eu](https://earth-system.usegalaxy.eu/). Go running on the subdomain page to try out these tools !
+Don't hesitate to give use some feedbacks and help if you think they should be improve. As said before they are quite new and need to be tested and approved.
+
+The final goal will be to have at least one workflow of tools to access, process and visualise data for each domain cited above. Hopefully even more than that ! 
+If you are interrested by earth system and environmental sciences and have tools that you think could fit right into earth-system.usegalaxy.eu, please contact us so we can try to work together on them ! The more the merrier ! 
+
+This work takes place within the cooperation of the Fair-Ease and EuroScienceGateway projects (to know more on the projects see the [blog](https://galaxyproject.org/news/2023-05-21-fair-ease-euro-science-gateway/)).
```
Comment (bgruening) [inline content/news/2023-10-17-earth-system/index.md:46]:
```suggestion
This work takes place within the cooperation of the FAIR-Ease and EuroScienceGateway projects (to know more on the projects see the [blog](https://galaxyproject.org/news/2023-05-21-fair-ease-euro-science-gateway/)).
```

**Review by bgruening — COMMENTED** [review]
Some small comments inline. Thanks for writing this.

**Comment by Marie59** [comment]
Thanks a lot for the review !!

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks!

**Pull request merged**